support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx

```diff
@@ -335,11 +363,15 @@ export function SummaryTsAndCs({
 	};
 
 	return (
-		<div css={[containerSummaryTsCs, cssOverrides]}>
-			{inSupport && copyTier1(contributionType)}
-			{inAllAccessDigital && copyTier2(contributionType, productKey)}
-			{inDigitalPlusPrint && copyTier3(contributionType, productKey, true)}
-			{inAdLite && copyTier3(contributionType, productKey, false)}
-		</div>
+		<>
+			{!inDigitalEdition && (
```

Comment:
Not sure we should be introducing product specific deviations from common patterns as we have done here for Digital Edition here? Why do we not include the Digital Edition's T&Cs as we do for each other product.

Reply:
I will create a trello card for an overhaul of Ts&Cs. Wonder whether it would be better to get this copy within the `productCatalog` itself  and access via `productKey` either from within a new generic Ts&Cs component or pass in the copy itself as you've mentioned. Probably needs repeating for the Summary Ts&Cs grey box too. Would you like me to make any further changes to this?

Reply:
@paul-daniel-dempsey good shout regarding getting this in the catalog, or at least extending the catalog's data with this info, we must do something similar with the Benefits list? I don't think it needs addressing here in this PR though.